react/features/stream-effects/virtual-background/JitsiStreamBackgroundEffect.js

```diff
@@ -130,13 +136,23 @@ export default class JitsiStreamBackgroundEffect {
             );
         }
         if (this._options.virtualBackground.backgroundType === VIRTUAL_BACKGROUND_TYPE.DESKTOP_SHARE) {
+
+            // save current context before applying transformations
+            this._outputCanvasCtx.save();
+
+            // flip the canvas and prevent mirror behaviour
```

Comment:
Are you flipping the screenshare here so that you would not have to use flip video for local video ? Does this fix the issue you were describing yesterday ?

Reply:
Yes. This fixes the issue that I described yesterday. My mistake was that yesterday I did not apply the save() and restore() functions.